src/framework/preferences/js/Panels.js

```diff
@@ -841,7 +841,7 @@ var fluid_2_0 = fluid_2_0 || {};
             theme: ["default", "bw", "wb", "by", "yb", "lgdg"]
         },
         markup: {
-            label: "<span class=\"fl-preview-A\">A</span><span class=\"fl-hidden-accessible\">%theme</span><div class=\"fl-crossout\"></div>"
+            label: "<span class=\"fl-preview-A\"></span><span class=\"fl-hidden-accessible\">%theme</span><div class=\"fl-crossout\"></div>"
```

Comment:
@jobara You're welcome.

I can't remember if, in the past, I suggested using aria-label as a solution in this case.  Maybe I did, and you had reasons for not using it.  FWIW, consider the following:

&lt;label for="radio" aria-label="Default"&gt;A&lt;/label&gt; &lt;input type="radio" id="radio"&gt;

With that markup:
- the name of the radio button is "Default".
- the text "Default" is not visually rendered.
- the text "A" is visually rendered.

Hope that helps.